tello Test/tello_web_server.py

```
@@ -828,39 +2176,113 @@ async def cors_middleware(request, handler):
+                    'Access-Control-Allow-Headers': 'Content-Type, Authorization',
                 }
             )
         
         try:
             response = await handler(request)
             response.headers['Access-Control-Allow-Origin'] = '*'  # 本番環境では特定のドメインに制限することを推奨
             response.headers['Access-Control-Allow-Methods'] = 'GET, POST, OPTIONS'
-            response.headers['Access-Control-Allow-Headers'] = 'Content-Type'
+            response.headers['Access-Control-Allow-Headers'] = 'Content-Type, Authorization'
             return response
+        except web.HTTPMethodNotAllowed as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)